spec/decorators/article_decorator_spec.rb

```diff
@@ -0,0 +1,32 @@
+require "rails_helper"
```

Comment:
these tests where in the `article_spec.rb` but they belong in a decorator specific test